src/RuleSet/Sets/PERCSRiskySet.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\RuleSet\Sets;
+
+use PhpCsFixer\RuleSet\AbstractRuleSetDescription;
+
+/**
+ * @internal
+ *
+ * @link https://www.php-fig.org/per/coding-style/
+ *
+ * This Ruleset always points to the latest version of PER-CS.
```

Comment:
I wonder if we can have test for that?
I imagine someone creating PERCS1.2, but forgetting to update definition of PERCS

Reply:
Off the cuff guess: A test that scans the folder for class names, filters for `PERCS`, orders alphabetically, and pops the first one?  Then use that to see if it resolves to the same name that `PERCSSet` points to?  Just spitballing.

Reply:
i would only replace first part, you can use https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/src/RuleSet/RuleSets.php#L55 instead of scanning the folder